changer de compte client

joannagb opened the pull request:
Suite aux mots saisis "changer de compte",  j'ai crée cette consigne en m'appuyant sur la consigne semblable, à savoir  'Comment vérifier quel compte client est enregistré sur ma liseuse TEA ?'  Cette dernière, reponds en fait déjà à la  question de changement de compte mais  l'intitulé ne l'indique pas.  J'aimerais avoir votre avis, sur le bien-fondé de cette nouvelle consigne. A la limite, on pourrait aussi passer par une éventuelle réformulation de la question existante ( mais qui toutefois,  pourrait la complexifier quelque peu)

joannagb committed:
Comment changer de compte client sur une liseuse?

Suite aux mots saisis "changer de compte",  j'ai crée cette consigne en m'appuyant sur la consigne semblable, à savoir  'Comment vérifier quel compte client est enregistré sur ma liseuse TEA ?'  Cette dernière, reponds en fait déjà à la  question de changement de compte mais  l'intitulé ne l'indique pas.  J'aimerais avoir votre avis, sur le bien-fondé de cette nouvelle consigne. A la limite, on pourrait aussi passer par une éventuelle réformulation de la question existante ( mais qui toutefois,  pourrait la complexifier quelque peu)

astranchet commented on _pages/FAQ-comptes.md:
```diff
@@ -58,9 +58,18 @@ Actuellement, votre liseuse ne peut se connecter qu'à un seul compte à la fois
 
 {% include feedback-button.html question="Comment vérifier quel compte client est entré sur ma liseuse ?" %}
 
+{% include anchor.html id=" changer de compte client de ma liseuse" label="Comment changer de compte client de ma liseuse TEA ?" %}
```
Un identifiant (le paramètre "id") doit être "technique" : sans espace, surtout au début, contrairement au label. Ici, l'identifiant pourrait être "changement-compte-liseuse"

joannagb replied:
Corrigé

astranchet commented on _pages/FAQ-comptes.md:
```diff
@@ -58,9 +58,18 @@ Actuellement, votre liseuse ne peut se connecter qu'à un seul compte à la fois
+Pour changer de compte client sur une liseuse TEA :
+- Aller dans « Paramètres »
+- Puis dans « Comptes et synchronisation »
+- Et enfin dans « Mon compte client »
```
Je serais d'avis d'écrire ces 3 lignes en une seule : Aller dans « Paramètres / Comptes et synchronisation / Mon compte client ».

joannagb replied:
Ouiiii moi aussi, c'était mon idée du debut aussi. Puis , j'ai vu la consigne existante sur la vérif compte liseuse  et  je me suis allignée sur la présentation. T'es ok pour que je  change cela dans les deux ?

astranchet replied:
Fait dans une autre PR la deuxième modification. Mais effectivement, ça manque d'harmonisation, parfois on fait en plusieurs phrases, parfois en une seule, parfois avec des "/". Il faudrait aussi croiser avec @LMargot

joannagb replied:
vu avec Lucile, modif faite dans la PR  : Harmonisation présentation chemin : Par/... #198

astranchet commented on _pages/FAQ-comptes.md:
```diff
@@ -58,9 +58,18 @@ Actuellement, votre liseuse ne peut se connecter qu'à un seul compte à la fois
+- Puis dans « Comptes et synchronisation »
+- Et enfin dans « Mon compte client »
+- Si vous voyez apparaître une icône de cadenas ouvert ainsi que la mention « Connexion » vous pouvez vous enregistrer en saisissant l' adresse mail de votre compte.
+- Si en revanche, vous voyez déjà una adresse mail ainsi que la mention « Se déconnecter » cela signifie qu'un compte client est déjà connecté. Dans ce cas, il faudra le déconnecter car actuellement, votre liseuse peut être connectée à un seul compte à la fois.
```
Je serais d'avis de simplifier aussi ces deux lignes en une seule : Cliquer sur « Se déconnecter » pour pouvoir vous connecter à un nouveau compte.

joannagb committed:
modif ID : "changement-compte-liseuse"

fait

joannagb committed:
chemin compacté sur 1 ligne : Par/ Cpt...  

Indication de chemin a été compactée sur 1 ligne au lieu de 3 précedemment, ce qui donne :
Aller dans « Paramètres / Comptes et synchronisation / Mon compte client

joannagb committed:
texte 'déconnexion' simplifié 

texte après modif d'AnneSo  : 
"Cliquez sur « Se déconnecter » pour pouvoir vous connecter à un nouveau compte."

astranchet commented on _pages/FAQ-comptes.md:
```diff
@@ -58,9 +58,16 @@ Actuellement, votre liseuse ne peut se connecter qu'à un seul compte à la fois
 
+Pour changer de compte client sur une liseuse TEA :
+- Aller dans « Paramètres / Comptes et synchronisation / Mon compte client ».
+- Si vous voyez apparaître une icône de cadenas ouvert ainsi que la mention « Connexion » vous pouvez vous enregistrer en saisissant l' adresse mail de votre compte.
```
Proposition de simplification du texte "Si vous voyez apparaître une icône" => "Si vous voyez"

astranchet commented on _pages/FAQ-comptes.md:
```diff
@@ -58,9 +58,16 @@ Actuellement, votre liseuse ne peut se connecter qu'à un seul compte à la fois
+Pour changer de compte client sur une liseuse TEA :
+- Aller dans « Paramètres / Comptes et synchronisation / Mon compte client ».
+- Si vous voyez apparaître une icône de cadenas ouvert ainsi que la mention « Connexion » vous pouvez vous enregistrer en saisissant l' adresse mail de votre compte.
+- Si en revanche, vous voyez déjà una adresse mail, cliquez sur « Se déconnecter » pour pouvoir vous connecter à un nouveau compte. Votre liseuse peut être connectée à un seul compte à la fois.
```
Proposition de simplification du texte 'Si en revanche, vous voyez déjà una adresse mail, ..." => "Si un compte est déjà connecté, ..."

astranchet reviewed (commented):
Je procède par itération en proposant des textes encore plus courts (et donc plus simple à lire)

LMargot commented:
Comme autre réponse à cette question il y a aussi l'utilisation des profils utilisateurs, mais je connais moins bien la fonctionnalité.

joannagb committed:
Textes simplifiés 

Textes simplifiés conformément aux remarques
